README.md

```diff
@@ -203,7 +203,11 @@ Install
     $ pip install --no-binary :all: confluent-kafka
 
 
-For source install, see *Prerequisites* below.
+For source install, see *Prerequisites* below.   
+
+To avoid error in installation like *no matches found: confluent-kafka[avro]*, use double quotes. As an example:
+
+    $ pip install "confluent-kafka[avro]"
```

Comment:
I think it is better to just change the "pip install ..avro]" line above to include quotes, people tend to cutnpaste the first command they see

Reply:
Sorry, this is my first time to change. Where should I change the line ?. In my forking repo ?

Reply:
Yes, undo your previous changes and modify the original `pip install confluent-kafka[avro]` line above to have quotes. Commit and push.